assembl/static2/js/app/components/debate/thread/post.jsx

```diff
@@ -30,12 +31,16 @@ const getFullLevelString = (fullLevel) => {
   );
 };
 
+// TODO we need a graphql query to retrieve all languages with native translation, see Python langstrings.LocaleLabel
```

Comment:
Half the job is already done. The discussion's preferred languages are pulled via grapqhl using `langstring.LocaleLabel` to get their proper names. Can use as a starting point if you wish. New GraphQL queries will be needed for per user preferences with all languages translated though.

Reply:
Yeah, I saw that in your branch. So I'm waiting for your work to be merged to work on it.